services/hapi/src/libs/sync-bps.js

```diff
@@ -3,7 +3,7 @@ const EosApi = require('eosjs-api')
 const massive = require('massive')
 const request = require('request-promise')
```

Comment:
@danazkari  I removed the directory that held the `update-bps` script that saved bps and bp.json to postgres and included it here. That script is being called by the daily cron job o we would need to update its path to this directory directory